exercises/etl/etl.jl

```diff
@@ -1,4 +1,4 @@
-function transform(input::AbstractArray)
+function transform(input::Dict)
```

Comment:
This should be an abstract type, i.e. `AbstractDict`

Reply:
Sounds good